File: AppSettingsManagerUI/src/app/services/bff/models/CreateSettingRequest.ts
export class CreateSettingRequest {
    SettingGroupId: string;
    Input: string;
    UserId: string;
    Password: string;

    constructor(userId: string, settingGroupId: string, input: string, password: string){
        this.UserId = userId;
        this.SettingGroupId = settingGroupId;
        this.Input = input;
        this.Password = password;
    }
}

File: AppSettingsManagerBff/AppSettingsManagerBff.Model/Requests/CreateSettingRequest.cs
using System.Text.Json.Nodes;

namespace AppSettingsManagerBff.Model.Requests;

public class CreateSettingRequest
{
    public string SettingGroupId { get; set; }
    public JsonNode Input { get; set; }
    public string UserId { get; set; }
    public string Password { get; set; }
}
